src/app/dashboard/experience-form/experience-form.component.html

```diff
@@ -0,0 +1,73 @@
+<form nz-form [formGroup]="experienceForm" >
+  <ng-container formArrayName="experiences">
+    <div *ngFor="let experience of experiences.controls; let idx = index" [formGroupName]="idx">
```

Comment:
Lets make its as compact as possiblle. Lets use
```<nz-divider nzText="Text"></nz-divider>``` this div is taking to much space.